source/autoSettingsUtils/driverSetting.py

```diff
@@ -37,6 +37,9 @@ class DriverSetting(AutoPropertyObject):
 	#: Set this to C{False} if L{AutoSettings} deals with loading and saving.
 	useConfig: bool = True
 
+	#: Type information for _get_configSpec
+	configSpec: ClassVar[str]

```

Comment:
I think this is a bit misleading, configSpec is an instance variable.

Reply:
I assume the problem with this line is that the dataclass annotation picks this up and generates stuff for it? Is there a way to get the dataclass to ignore it?

Reply:
Yes, the way for dataclasses to ignore this is exactly this.

> If a field is a ClassVar, it is excluded from consideration as a field and is ignored by the dataclass mechanisms. Such ClassVar pseudo-fields are not returned by the module-level fields() function.

See https://docs.python.org/3/library/dataclasses.html

I however read the docs about ClassVar and indeed, it is really meant to annotate class variables. Bleg.